### PR TITLE
trim input value

### PR DESCRIPTION
**Pull request by Bavon101**
This PR changes involve the trimming of `input value` in order to prevent a user from adding an empty task.

### CODE REVIEW

File: src/modules/tasks.js [hunk 1]
```diff
@@ -53,7 +53,8 @@ export default class Tasks {
 
   taskExists = (d) => {
     d = d.toLowerCase().trim();
```
Comment (mnekx) [inline src/modules/tasks.js:55]: According to W3 javascript best practices, the 'd' could either be given a default value [(d='')] OR it is tested inside the function for 'undefined' [if(d !== undefined) {// function code inside}]. This is due to the fact that If a function is called with a missing argument, the value of the missing argument is set to "undefined". Undefined values can break your code. It is a good habit to assign default values to arguments.

Comment (Bavon101) [reply src/modules/tasks.js:55]: Hello @mnekx ,
Thanks for the review, yes that's is the best practice, but it implies how the function is called, where the function is called the value of `d` can never be `null` or `undefined`.
Please take a look at where and how the function is called.

**Review by mnekx — APPROVED** [review]
Congratulations. Your project is complete. There is nothing left to be done other than to merge it. 💯  
- [x] Javascript Best practices
- [x]  HTML and CSS best practices
- [x] DRY, KISS and YAGNI rules

**Comment by Bavon101** [comment]
> Congratulations. Your project is complete. There is nothing left to be done other than to merge it. 💯
> 
> * [x]  Javascript Best practices
> * [x]   HTML and CSS best practices
> * [x]  DRY, KISS and YAGNI rules

Thanks for the review, the requested changes have been implemented 🐱‍🏍✔

**Pull request merged**